openapi.yaml

```
@@ -6517,6 +6517,17 @@ paths:
                     description: >
                       An array of tags applied to the Kubernetes cluster. Tags are for organizational purposes only.
                       To delete a tag, exclude it from your `tags` array.
+                k8s_version:
+                    type: string
+                    description: >
+                      The desired upgraded Kubernetes version for this Kubernetes cluster in the format of
+                      &lt;major&gt;.&lt;minor&gt;. New and recycled Nodes in this cluster will be installed with the
+                      latest available patch for the Cluster's upgraded Kubernetes version.
```

Can we remove the instances of the word "upgraded" in this first paragraph? I think that makes more sense here, since this is representing the desired state of the cluster and that may not necessarily be an upgrade. For example, you can hit this endpoint giving the same version you are currently on.
```suggestion
                      The desired Kubernetes version for this Kubernetes cluster in the format of
                      &lt;major&gt;.&lt;minor&gt;. New and recycled Nodes in this cluster will be installed with the
                      latest available patch for the Cluster's Kubernetes version.
```